core/pom.xml

```diff
@@ -49,7 +49,7 @@
     <parent>
         <groupId>org.jahia.modules</groupId>
         <artifactId>module-manager-parent</artifactId>
-        <version>2.2.0-SNAPSHOT</version>
+        <version>2.2.1-SNAPSHOT</version>
```

Comment:
@rknj just want to run this by you, but I've bumped up the micro version for 3 pom.xml in this repo for the change. Just want to make sure that makes sense.

Reply:
I don't understand why you need to change the version ? 2.2.0 has not been released, this change could be part of it

Reply:
I agree we can keep 2.2.0. @gflores-jahia please note that the last digit is only for patches (see https://confluence.jahia.com/display/PR/Versioning#Versioning-Patch)

Reply:
Version bump has been reverted. Keeping to 2.2.0-SNAPSHOT.